### DISCUSSION TITLE
How to copy cell content from table to Excel with formatting (bold for example)?

### DISCUSSION THREAD

**Discussion by poltor** [glideapps/glide-data-grid#357] [Q&A] [answered]
How to copy cell content from table to Excel with formatting (bold for example)?

**Comment by jassmith** [comment, marked as answer]
Excel encodes this information as CSS styles applied to table elements in the `text/html` portion of the paste buffer. The exact syntax changes from version of excel to version of excel.

You would need to capture the paste event and translate it to something GDG understands in the theme override.

**Reply by poltor** [reply]
Thank you for quick reaction! I can try this way a little bit later then added comment here
